logfire/_internal/exporters/console.py

```diff
@@ -251,6 +251,18 @@ def _details_parts(self, span: Record, indent_str: str) -> TextParts:
         else:
             return []
 
+    def is_schema_valid(self, schema: dict[str, Any], value: Any) -> bool:
+        """Check if the schema is valid for the value."""
+        schema_type = schema.get('type') if schema else None
+        if schema_type:
+            if value is None:
+                return False
+            elif isinstance(value, dict) and schema_type != 'object':
```

Comment:
i think `if schema_type == 'object' and not isinstance(value, dict):` would work better

Reply:
makes sense I've made the changes